index.html

```diff
@@ -91,8 +91,8 @@ <h4 class="my-0 font-weight-normal">Windows</h4>
             <i class="fab fa-windows fa-7x icons-dowloads"></i>
             <ul class="list-unstyled mt-3 mb-4">
               <b>System Requirements:</b>
-              <li>OS: Windos 7 or above</li> 
-              <li>Processor: x86 or x64 bits </li>
+              <li>OS: Windows 7 or above</li> 
+              <li>Processor: 32 or 64 bits </li>
```

Comment:
x86 and x64 are more accurate than 32-bit and 64-bit, since they also mention the required architecture (x86, in contrast to ARM).

LibreSprite can be compiled for ARM, but official releases are x86-only.